Gemfile

```diff
@@ -3,22 +3,11 @@ ruby "3.1.0"
 
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
-gem 'rails', '~> 6.1'
+gem 'rails', '~> 7.0'
 gem 'puma', '~> 5'
 gem 'webpacker', '~> 5'
 gem 'turbolinks', '~> 5'
-# Use SCSS for stylesheets
-# gem 'sass-rails', '>= 6'
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-# gem 'jbuilder', '~> 2.7'
-# Use Redis adapter to run Action Cable in production
-# gem 'redis', '~> 4.0'
-# Use Active Model has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
-
-# Use Active Storage variant
-# gem 'image_processing', '~> 1.2'
+gem 'sprockets-rails', :require => 'sprockets/railtie'
```

Comment:
https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#sprockets-is-now-an-optional-dependency

Kept as-is as part of the upgrade